docs/pages/repo/docs/core-concepts/caching/what-to-cache.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: What to Cache?
+description: The configuration of Turborepo `outputs`
+---
+
+import Callout from "../../../../../components/Callout";
+
+# What to Cache?
+
+By default, the console output (also known as `stdout`) from all tasks is _always_ captured by Turborepo. If your task does not emit any files (e.g. unit tests with Jest) you can omit `outputs`. Beyond that, you will generally need to specify which files on disk you need to capture (and restore!) in order to be able to run subsequent dependent tasks.
```

Comment:
"capture (and restore!)" makes sense to me because I know what it's referring to, but the user is not the one doing the capturing or restoring. The capturing and restoring is also not guaranteed to happen (they could just stay unchanged where the task wrote them), so might be the wrong level of detail for this point. Maybe there's another way to word this? 
"...specify which output files are required to be able to run ..."
"specify which of your task's output files are used by dependent tasks"
something else?